flink-libraries/flink-table/src/main/scala/org/apache/flink/api/table/expressions/call.scala

```diff
@@ -18,85 +18,26 @@
 package org.apache.flink.api.table.expressions
 
 import org.apache.calcite.rex.RexNode
-import org.apache.calcite.sql.SqlOperator
-import org.apache.calcite.sql.fun.SqlStdOperatorTable
 import org.apache.calcite.tools.RelBuilder
+import org.apache.flink.api.table.validate.{ExprValidationResult, ValidationFailure}
 
 /**
   * General expression for unresolved function calls. The function can be a built-in
   * scalar function or a user-defined scalar function.
   */
-case class Call(functionName: String, args: Expression*) extends Expression {
+case class Call(functionName: String, args: Seq[Expression]) extends Expression {
```

Comment:
The reason why I used var-args here was that it is easier to be used in the API once custom functions are possible: `.filter(Call("MYFUNC", 'f1, 'f2))`. But we can also find an other way later.